server/bleep/src/webserver/answer.rs

```diff
@@ -286,19 +286,27 @@ async fn _handle(
     }
 
     let answer_bearer = if app.env.allow(Feature::GithubDeviceFlow) {
-        let Some(cred) = app.credentials.get(&Backend::Github) else {
+        let Some(cred) = app.credentials.github() else {
```

Comment:
Separate story, but it would be nice to have a way of bypassing this logic for debugging.